db_helper.py

```diff
@@ -1,141 +1,140 @@
+        today = date.today()
         last_mix = self.get_last_mix()
-        aux_date = None
         if last_mix:
-            for row in last_mix:
-                aux_date = row[1].split(' ')[0]
-            last_mix_day = datetime.strptime(aux_date, "%Y-%m-%d")
-            if today.date() == last_mix_day.date():
+            if today == last_mix.mix_date:
                 return True
             else:
                 return False
         else:
             return False
 
     def create_mix(self, description):
-        msg = None
-        print(self.mix_exist())
         if not self.mix_exist():
-            stmt = "INSERT INTO mix (description) VALUES (?)"
+            self.session.add(Mix(description=description,
+                    mix_date=date.today()))
         elif self.mix_today():
             msg = 'Ya ha sido creado'
-        else:
-            stmt = "INSERT INTO mix (description) VALUES (?)"
         try:
-            self.conn.execute(stmt, (description,))
-            self.conn.commit()
+            self.session.commit()
             msg = 'Mix creado correctamente'
-        except:
+        except Exception:
             pass
         return msg
 
     def add_item(self, id_telegram, first_name, alias):
         user = self.get_user(id_telegram)
-        user_id = None
-        for row in user:
-            user_id = row[0]
-        if user_id is None:
+        if not user:
             self.user_create(id_telegram, first_name, alias)
             user = self.get_user(id_telegram)
-            for row in user:
-                user_id = row[0]
         last_mix = self.get_last_mix()
-        mix_id = None
-        for row in last_mix:
-            mix_id = row[0]
-        if self.user_in_mix(user_id, mix_id) == 0:
-            stmt = "INSERT INTO mix_user (user_id, mix_id) VALUES (?, ?)"
-            args = (user_id, mix_id,)
-            self.conn.execute(stmt, args)
-            self.conn.commit()
+        if last_mix and not self.user_in_mix(user.id, last_mix.id):
+            self.session.add(MixUser(user_id=user.id,
+                    mix_id=last_mix.id))
+            return self.session.commit()
+        return False
 
     def delete_item(self, id_telegram):
         last_mix = self.get_last_mix()
         user = self.get_user(id_telegram)
-        if user:
-            user_id = None
-            mix_id = None
-            for row in user:
-                user_id = row[0]
-            for row in last_mix:
-                mix_id = row[0]
-            if self.user_in_mix(user_id, mix_id) == 1:
-                stmt = "DELETE FROM mix_user WHERE user_id = (?) AND mix_id = (?)"
-                args = (user_id, mix_id,)
-                self.conn.execute(stmt, args)
-                self.conn.commit()
+        if user and self.user_in_mix(user.id, last_mix.id):
+            self.session.query(MixUser).filter(MixUser.user_id == user.id,
+                MixUser.mix_id == last_mix.id).delete()
```

Comment:
it would be awesome , if each time a user that is a player (within 10 first) is removed ,the 11 player who is in the reserve and is going to replace that player , is notified by the bot(@username) , and the new table(get_mix) is called displaying the list on telegram.

Reply:
Extended the /out message to display if a "suplente" goes to main team (needs test)